Source/WebKit/UIProcess/Cocoa/UserMediaCaptureManagerProxy.h

```diff
@@ -70,8 +70,10 @@ class UserMediaCaptureManagerProxy : public IPC::MessageReceiver {
         virtual void addMessageReceiver(IPC::ReceiverName, IPC::MessageReceiver&) = 0;
         virtual void removeMessageReceiver(IPC::ReceiverName) = 0;
         virtual IPC::Connection& connection() = 0;
```

Comment:
Can we remove this one?

Reply:
We shouldn't. It is still useful when calling trivial functions which do not require a smart pointer.